utils/call-gen/src/lib.rs

```diff
@@ -44,8 +45,12 @@ pub enum GearCall {
     UploadCode(UploadCodeArgs),
 }
 
-pub fn generate_gear_program<Rng: CallGenRng>(seed: u64) -> Vec<u8> {
+/// Function generates WASM-binary of a Gear program with the
+/// specified `seed`. `programs` may specify addresses which
+/// can be used for send-calls.
+pub fn generate_gear_program<Rng: CallGenRng>(programs: Vec<ProgramId>, seed: u64) -> Vec<u8> {
```

Comment:
```suggestion
pub fn generate_gear_program<Rng: CallGenRng>(seed: u64, programs: Vec<ProgramId>) -> Vec<u8> {
```
For me seed seems more basic and important here